### PR TITLE
extract references from yaml header into separate, centralized file

### PR DESCRIPTION
**Pull request by d-morrison**
I'm starting to clean up the `tutorial.Rmd` file; here I extracted the article references from the header into a separate file. @kristinawlai, please take a look at what I've done here; we'll want to do the same thing other places as well.

I also removed the handwritten section numbers; it's more sustainable to have Rmarkdown number them for us.

### CODE REVIEW

File: vignettes/tutorial.Rmd
Comment (kristinawlai) [inline vignettes/tutorial.Rmd:1]: @d-morrison A small point... do we still want numbered sections here and throughout the vignettes? If so, we would do this by adding "number_sections: true" to the YAML section, correct? (I'm learning!)

**Review by kristinawlai — CHANGES_REQUESTED** [review]
All looks good except for my question about numbered sections.

File: vignettes/tutorial.Rmd
Comment (d-morrison) [reply vignettes/tutorial.Rmd:1]: @kristinawlai correct - added! (although in PR #15, we switch to a different output format, `bookdown::html_document2`, which adds numbering by default).

Comment (kristinawlai) [reply vignettes/tutorial.Rmd:1]: @d-morrison Ah okay I was just going to look that up. Thanks!

**Review by kristinawlai — APPROVED** [review]

**Pull request merged**